### DISCUSSION TITLE
v7.3 detected as malware

### DISCUSSION THREAD

**Discussion by pptsk** [avrdudes/avrdude#1736] [Q&A]
Hi,
I am using avrdudess https://github.com/ZakKemble/AVRDUDESS which has included avrdude.exe and avrdude.conf.
New version of avrdudess(2.17) contain also new avrdude 7.3 but after installing my Antivirus software Iobit MalWareFighter detecting  avrdude as malware:  avrdude.exe Gen: Variant.Babar.452770
If I replace with old version where is avrdude 7.2 everything works fine.
What is wrong?
Thank you

**Comment by dl8dtl** [comment]
> What is wrong?

Most likely your antivirus software.

However, it's not clear exactly where the binary is from you installed. If you want to be sure, you can install one of our official releases instead. That's the only way we (as a project) could guarantee you anything. For any 3rd-party compilation, you'd have to ask your vendor.

**Comment by mcuee** [comment]
I do not see this in my Windows 10 and Windows 11 system.

**Comment by Ian-Johnston** [comment]
BitDefender reports same issue with the downloaded ZIP and EXE via Zak's site.
If it's a false positive then it's very simple for Zak to contact BitDefender and have the file analyzed and the threat pulled from their AV database.
I have tried contact Zak direct......we'll see if he replies and/or fixes the issue.
